openff/toolkit/utils/rdkit_wrapper.py

```diff
@@ -401,8 +451,16 @@ def _polymer_openmm_topology_to_rdmol(
                     for atom_i, j in zip(ref.GetAtoms(), match):
                         atom_j = mol.GetAtomWithIdx(j)
                         # copy over chirality
+                        # TODO: This is most likely wrong. This will copy over CW/CCW tags but there's no
+                        #  checking that the substituent order in the molecule and substructure are the same,
+                        #  so this is basically random assignment. Thankfully Molecule.from_polymer_pdb and
+                        #  Topology.from_pdb both overwrite this assignment based on 3D coordinates. The only
+                        #  way I can currently think of doing this right would be to arbitrarily assign CW,
+                        #  then try to run a strict substructure match on it, and if that fails then assign
+                        #  CCW. This would be monstrously inefficient and all it would let us do is compare
+                        #  the substructure stereo to 3D stereo, which is of dubious value anyway.
```

Comment:
Big blocker here, this basically summarizes it. This will need a lot more work and I don't think it's worth holding up the release for.